lib/filter_nb.c

```diff
@@ -255,6 +257,12 @@ lib_access_list_entry_action_modify(struct nb_cb_modify_args *args)
 	else
 		f->type = FILTER_DENY;
 
+	/* Run hook function. */
```

Comment:
Can you move this into a `static` function, please? Same applies for all other duplicated code.

We'd have two functions: one for add another for delete or you can use a boolean parameter to toggle between them.

Reply:
Thanks for the review. 
The changes are done.